money_honey_cr.py

```diff
@@ -0,0 +1,147 @@
+import json
+import sys
+import getopt
+from decimal import Decimal, InvalidOperation
+from random import randint
+
+from actual_currencies import CurrenciesChanger
+
+
+class MoneyHoney:
+    """
+    I wasn't sure if it's allowed to use external libraries so I didn't use any.
+    Estimated time of coding 4MD.
+    """
+    def __init__(self):
+        pass
+
+    def arguments_handler(self):
```

Comment:
Yea reworked in [#7](https://github.com/ZlatyChlapec/MoneyHoney/commit/27c422c0ff89951c23ff16acb3dd78deae4de258).